### DISCUSSION TITLE
You were last verified as a student on It is not necessary for you to reverify at this time. There may be a wait period between verification and access to academic benefits

### DISCUSSION THREAD

**Discussion by ghost** [community/community#146513] [GitHub Education]
### Hi everyone,

I want to reapply after my student package expires, but when I apply, I get a message that says "Hi, cgbb0!" You were last verified as a student on Nov 25, 2023. It is not necessary for you to reverify at this time. There may be a wait period between verification and access to academic benefits.”。 I've been waiting for 10 days and still nothing has changed.

**Comment by davevad93** [comment]
Hi @cgbb0 ,

Technically, you don't need to re-apply right now, because you were last verified as a student on Nov 25, 2023, and student benefits last for 2 years, so you should re-apply on Nov 25, 2025.

**Reply by ghost** [reply]
I activated the student offer for Azure last year using the GitHub Student Benefit Pack, but it has now expired. When I re-tried activating the student offer for Azure with the GitHub Student Benefit Pack, Azure prompted me that I didn't meet the requirements.
![image](https://github.com/user-attachments/assets/a1fe4184-f502-494a-a1c1-9faaa249594b)

**Reply by davevad93** [reply]
Hey @cgbb0 , that looks like a problem with Azure because it has happened in the past to other verified students as well (see this #135739), contact support by opening a ticket [here](https://support.github.com/contact/education), select "`I am a verified student having trouble redeeming my Global Campus offers`" and explain your issue.

**Reply by ghost** [reply]
Thank you for the offer, I'll give it a try

**Reply by davevad93** [reply]
You're welcome @cgbb0 👍
